### PR TITLE
fix(protocol): update nc (netcat) pkg name and add error handling

### PR DESCRIPTION
**Pull request by kimo-ice**

### CODE REVIEW

**Review by davidtaikocha — APPROVED** [review]

File: .github/workflows/protocol.yml [hunk 1]
```diff
@@ -20,7 +20,8 @@ jobs:
           access_token: ${{ github.token }}
 
       - name: Prepare environment
-        run: sudo apt-get update && sudo apt-get install -y git nc
+        continue-on-error: true
+        run: sudo apt-get update && sudo apt-get install -y git netcat
```
Comment (mask-pp) [inline .github/workflows/protocol.yml:24]:
```suggestion
        run: sudo apt-get update && sudo apt-get install -y git netcat-traditonal
```

Comment (mask-pp) [reply .github/workflows/protocol.yml:24]: I'm not sure if it's correct to install `netcat` in `arc-runner-set`, but if it's `apt-get` cmd source we need to install `netcat-traditonal`.

Comment (kimo-ice) [reply .github/workflows/protocol.yml:24]: netcat-openbsd (netcat) is the newer 'more powerful' version. It should work fine.

**Review by RogerLamTd — APPROVED** [review]

File: .github/workflows/protocol.yml (hunk 1 above)
Comment (mask-pp) [reply .github/workflows/protocol.yml:24]: Better way, don't need to install nc, just replace line58 like this `until cast chain-id --rpc-url "htp://localhost:8545" 2> /dev/null; do` is ok .

**Review by mask-pp — APPROVED** [review]

**Pull request merged**